### ISSUE TITLE
Indent rule doesn't cover multiline VariableDeclarations

### ISSUE THREAD

**Issue by Trott** [eslint/eslint#6911]
**What version of ESLint are you using?**
v3.3.0

**What parser (default, Babel-ESLint, etc.) are you using?**
default

**Please show your full configuration:**
n/a

**What did you do? Please include the actual source code causing the issue.**
Indent rule ignores multiline VariableDeclarations like this:

``` js
var foo = 'bar' +
'baz';
```

**What did you expect to happen?**

I expect ESLint to flag that as a problem.

**What actually happened? Please include the actual, raw output from ESLint.**

It did not flag it as a problem.

This is a subset of https://github.com/eslint/eslint/issues/1801. I'm opening this because I intend to submit a fix for this specific issue soon...

**Comment by gyandeeps** [comment]
I dont know why but i think we should use the indentation option set for variables here too, what am i missing?

Like i consider both cases kinda of the same

case 1:
```js
var a = 1,
    b = 2;
```

case 2:
```js
var a = "test" +
    "temp";
```

**Comment by Trott** [comment]
@gyandeeps But might you not want this?

```js
var a = 2,
    b = "foo",
    c = "bar" +
        "bip",
    d = 3;
    e = "one long string" +
        "another long string" +
        "one more really long strong",
    f = 4;

**Comment by gyandeeps** [comment]
based on my view, this looks even better. havent thought about this. :smile:

**Comment by Trott** [comment]
Another reason to not use the setting for variables but to instead have a separate one for BinaryExpressions: The idea is to expand where this applies to that it also covers other multi-line BinaryExpressions such as ones occuring in variable assignments, function arguments, conditionals, etc.

**Comment by not-an-aardvark** [comment]
Closing this issue in favor of https://github.com/eslint/eslint/issues/8443.
